spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/GatewayApiController.java

```diff
@@ -51,4 +65,27 @@ public ApiKey findApiKeyEntityByApiKeyValue(@RequestParam("keyValue") String api
     public ServiceApi findServiceByApiKeyValue(@RequestParam("apiKeyValue") String apiKeyValue) {
         return serviceApiRepository.findByApiKeyValue(apiKeyValue);
     }
+
+    @GetMapping( "/api/gateways/{gatewayId}/apiKeys")
+    public ResponseEntity<List<ApiKey>> findApiKeyListByGatewayId(@PathVariable("gatewayId") String gatewayId) {
+        Proxy proxy = null;
+        List<ApiKey> apiKeyList;
+        String apiKeyListCreationFailedMessage = "Creation of API key list for requesting gateway failed. " +
```

Comment:
It does not make sense to define an error string on each call before we even know if there _is_ an error. This should either be defined as constant or it should be defined in the catch block (when we actually know that there was an error).

Reply:
The String is removed now.